### PR TITLE
Code generation for feature columns in alps submitter

### PR DESCRIPTION
**Pull request by uuleon**

### CODE REVIEW

File: sql/alps_submitter.go [hunk 1]
```diff
@@ -46,6 +51,32 @@ type CrossColumn struct {
...
 
+func (nc *NumericColumn) GenerateCode() string {
+	return fmt.Sprintf("tf.feature_column.numeric_column(\"%s\", shape=(%d,))", nc.Key, nc.Shape)
+}
+
+func (bc *BucketColumn) GenerateCode() string {
+	return fmt.Sprintf(
+		"tf.feature_column.bucketized_column(%s, boundaries=%s)",
+		bc.SourceColumn.GenerateCode(),
+		strings.Join(strings.Split(fmt.Sprint(bc.Boundaries), " "), ","))
```
Comment (tonyyang-svail) [inline sql/alps_submitter.go:62]: Why not just `fmt.Sprint(bc.Boundaries)`?

File: sql/alps_submitter_test.go [hunk 2]
```diff
@@ -64,13 +66,7 @@ func TestColumnResolve(t *testing.T) {
 	a.Equal(10, nl2.Shape)
 	a.Equal([]int{1, 10}, bl.Boundaries)
 
-	a.Equal("NumericColumn", getFeatureColumnType(cl.Keys[1]))
```
Comment (tonyyang-svail) [inline sql/alps_submitter_test.go:67]: why removing the following assertion?

File: sql/alps_submitter.go (hunk 1 above)
Comment (uuleon) [reply sql/alps_submitter.go:62]: The `fmt.Sprint` results a string with space separator which I want it to be a comma .

File: sql/alps_submitter_test.go (hunk 2 above)
Comment (uuleon) [reply sql/alps_submitter_test.go:67]: That is a previous mistake. All keys of `cross` must be either string or categorical column except HashedCategoricalColumn, given `NumericColumn` which is not supported.

File: sql/alps_submitter_test.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ const (
 		WITH n_classes = 3 
 		COLUMN 
 			DENSE(c2, 5, comma), 
-			cross([BUCKET(NUMERIC(c1, 10), [1, 10]), NUMERIC(c4, 9), c5], 20) 
+			cross([BUCKET(NUMERIC(c1, 10), [1, 10]), c5], 20)
```
Comment (tonyyang-svail) [inline sql/alps_submitter_test.go:15]: Should we change `cross` to `CROSS` to make the feature column naming consistent?

**Review by tonyyang-svail — APPROVED** [review]

File: sql/alps_submitter_test.go (hunk 3 above)
Comment (uuleon) [reply sql/alps_submitter_test.go:15]: The purpose of this is to check if resolver can handle lowercase correctly.

**Pull request merged**